features/lima/generate-manifest.py

```diff
@@ -0,0 +1,82 @@
+import argparse
```

Comment:
```suggestion
#!/usr/bin/env python3
import argparse
```
This allows convinient and (subjectively)  lazy way of calling this helper via `./features/lima/generate-manifest.py`

cant do it in web-ui, but a `chmod +x features/lima/generate-manifest.py` would then also be required.